lib/API/DX/Device.cpp

```diff
@@ -518,6 +528,8 @@ class DXDevice : public offloadtest::Device {
                                "Failed to acquire UAV data pointer."))
       return Err;
     memcpy(ResDataPtr, R.BufferPtr->Data.get(), R.size());
+    if (R.HasCounter)
+      memcpy((char*)ResDataPtr + getUAVBufferCounterOffset(R), &R.BufferPtr->Counter, sizeof(uint32_t));
```

Comment:
This doesn't actually work right now. The value isn't filled in by the `memcpy` for some reason. I also tried using `memset`  with an arbitrary value which also didn't seem to help. 

I'm hoping that anyone reviewing can suggest ideas for resolution!

Reply:
Discussed offline with a few folks and this is WAI. The counter is automatically zeroed out so I should just delete this and the support from the yaml